src/notifications.c

```diff
@@ -152,6 +154,7 @@ int HashNotificationCallback(RedisModuleCtx *ctx, int type, const char *event,
     case del_cmd:
     case set_cmd:
     case trimmed_cmd:
+    case hexpire_cmd:

```

Comment:
Not sure this is the only update that needs to be done.
Key expiration is key deletion, but a field expiration could be either a key deletion (if last field expired) or only a field deletion so need to also look at the handling of `hdel_cmd`